### DISCUSSION TITLE
If you use fyne to develop a mobile app, what component of fyne does the bottom column of the app correspond to?

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4255] [Q&A] [answered]
# this Effect

![WX20230916-121155](https://github.com/fyne-io/fyne/assets/1819457/f9a4a60a-1c5d-444d-b876-3da58b9dcfd3)

**Comment by Jacalz** [comment, marked as answer]
Tab bar (apptabs or doctabs) with only icons and location as the bottom is the closest thing

**Reply by andydotxyz** [reply]
Definitely app tabs, that's basically what it was built for (though of course it's not mobile specific).
It will automatically go to the shorter edge on a mobile device so you don't need to configure anything.
